connector-packager/connector_packager/jar_jdk_signer.py

```diff
@@ -34,7 +38,7 @@ def validate_signing_input(input_dir, taco_name, alias, keystore):
     return True
 
 
-def get_user_pwd(alias):
+def get_user_pwd(alias: str) -> Tuple[bytes, bytes]:
```

Comment:
What is **Tuple[bytes, bytes]**? List of (byte, byte) tuples?

Reply:
Python is doing something unpython here, I think. It means it's returning a `(bytes, bytes)` tuple. A list of tuples would be `List[Tuple[bytes, bytes]]`. Yucky, right?

Reply:
Yucky, but oddly pythonic in the sense that I could read it